simulate.py

```diff
@@ -11,56 +11,64 @@
 @click.command()
 @click.option("-c", "--config", "config_file", envvar="CONFIG_PATH", type=click.Path(exists=True), required=True,
               help="configuration file for this simulation")
-@click.option("-s", "--scenario", type=str, required=True,
-              help="the scenario to run for this simulation")
-@click.option("-n", "--nsim", type=int, required=True, default=1000, show_default=True,
-              help="the # of model runs")
-@click.option("-j", "--jobs", type=int, default=multiprocessing.cpu_count(), show_default=True,
+@click.option("-s", "--scenario", "scenarios", type=str, default=[], multiple=True,
+              help="override the scenario(s) run for this simulation [supports multiple scenarios: `-s Wuhan -s None`]")
+@click.option("-n", "--nsim", type=click.IntRange(min=1),
+              help="override the # of simulation runs in the config file")
+@click.option("-j", "--jobs", type=click.IntRange(min=1),
+              default=multiprocessing.cpu_count(), show_default=True,
               help="the parallelization factor")
 @click.option("--interactive/--batch", default=False,
               help="run in interactive or batch mode [default: batch]")
 @click.option("--write-csv/--no-write-csv", default=True, show_default=True,
               help="write CSV output at end of simulation")
-def simulate(config_file, scenario, nsim, jobs, interactive, write_csv):
+def simulate(config_file, scenarios, nsim, jobs, interactive, write_csv):
     config.set_file(config_file)
 
     spatial_config = config["spatial_setup"]
     spatial_base_path = pathlib.Path(spatial_config["base_path"].get())
 
     interventions_base_path = pathlib.Path(config["interventions"]["scripts_path"].get())
-    script_npi = interventions_base_path / (scenario + ".R")
-    if not script_npi.exists():
-        raise click.BadParameter(f"NPI scenario file [{script_npi}] not found")
+    if not scenarios:
+        scenarios = config["interventions"]["scenarios"].as_str_seq()
```

Comment:
this will work even if the "scenarios" section looks like this
```{yml}
scenario1:
  param1: a, b, c
  param2: e,f,g
scenario2:
  param1: a,b,c
  param2: h,i,j
```
right?

Reply:
no, not yet.

i can add that, but are we moving the npis to python? or are these params passed to R via Rpy?

Reply:
in the case we provide parameters in this way, we would be passing them to python

Reply:
but development on that is in a different branch, so if there's things specific to that case, we should commit those in that branch

Reply:
ok, let's merge this in, we can then add scenario-specific config on the other branch